handlers/schemas/UploadVOD.yaml

```diff
@@ -31,8 +31,11 @@ properties:
         type: "integer"
       end_time:
         type: "integer"
+      playback_id:
+        type: "string"
     required:
       -  "start_time"
+      -  "playback_id"
```

Comment:
I don't think we should make this required (if we're aiming for non-Studio Catalyst users in the theoretical future)

Reply:
Done